src/authorizationProvider.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import authService from "./services/authService";
+
+const refreshToken = 'RefreshToken';
+const accessToken = 'AccessToken';
+const AuthorizationContext = React.createContext();
+class AuthorizationProvider extends Component {
+    state = {
+        loggedIn: null,
+        userFirstName: null,
+        userLastName: null,
+        userRole: null
```

Comment:
moze mat user viacero roli?

Reply:
pouzivatel moze byt: user, shelter admin alebo admin.. takto som to dostal na zaciatku